server/server.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import { json, urlencoded } from 'body-parser';
+// import swaggerUi from 'swagger-ui-express';
+import userRoute from './routes/users';
+// import loanRoute from './routes/loan';
```

Comment:
Remove the code that you have commented out please.

Reply:
Okay am going to remove them